lib/optimize/InnerGraphPlugin.js

```diff
@@ -140,6 +144,15 @@ class InnerGraphPlugin {
 									}
 								}
 							}
+							if (
+								statement.type === "ExpressionStatement" &&
+								parser.isPure(
+									statement.expression,
+									parser.prevStatement ? parser.prevStatement.end : 0
+								)
+							) {
+								pureExpressionStatements.add(statement);
+							}
```

Comment:
As there is no need to tag any symbol this part can be moved into `parser.hooks.statement` eliminating `pureExpressionStatements`

Reply:
ah, yes, thanks.